src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -124,21 +151,27 @@ const NavigationHeaderDropdownMenu = ({
         <button
           aria-controls={menuId}
           aria-expanded={isOpen}
-          className={s.activator}
+          aria-label={accessibleLabel}
+          className={classNames(s.activator, buttonClassName)}
           onClick={handleClick}
           onKeyDown={handleKeyDown}
           onMouseEnter={handleMouseEnter}
           ref={activatorButtonRef}
         >
-          <Text
-            asElement="span"
-            className={s.activatorText}
-            size={200}
-            weight="medium"
-          >
-            {label}
-          </Text>
-          <IconChevronDown16 className={s.activatorIcon} />
+          {leadingIcon && (
```

Comment:
[Food for thought](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) on using `&&` versus ternaries for rendering.

Reply:
I can update this to not use `&&`, but ternaries within JSX that span over multiple lines are hard to read. I think the two other options noted there are good though!
- Defining a variable via `if` statements before the `return` and passing it in the JSX
- Returning `null` early if the component can't render without a prop or computed value

Reply:
Adjusted in: https://github.com/hashicorp/dev-portal/pull/280/commits/c9ef94c6c44c9a222fcbede2da75af77ce681d97 ✅